modules/authentication/client-react/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@gqlapp/authentication-client-react",
+  "version": "1.0.0",
```

Comment:
Please check what version is used in other modules nearby and do the same

Reply:
Done